docs/blog/2019-01-29-themes-update-child-theming-and-component-shadowing/index.md

```diff
@@ -0,0 +1,131 @@
+---
+title: Themes Update: Child Theming and Component Shadowing
+date: 2018-01-26
+author: Chris Biscardi
+excerpt: "Making it easier to abstract themes into reusable modules"
+tags: ["themes", "architecture"]
+---
+
+> If you aren't familiar with Gatsby themes yet, then check out [Introducing
+> Gatsby
+> Themes](https://www.gatsbyjs.org/blog/2018-11-11-introducing-gatsby-themes/)
+> for a written introduction and [my Gatsby Days
+> talk](https://www.youtube.com/watch?v=wX84vXBpMR8) for a more audio/visual
+> approach.
+
+# Child Theming
+
+We've merged [a PR](https://github.com/gatsbyjs/gatsby/pull/10787) into Gatsby
+core to support Child theming. Child theming is an extension of the core theming
+algorithm to support a "parent/child" style relationship where child themes can
+rely on parent themes in the same way sites rely on themes. This means you can
+now use the `__experimentalThemes` gatsby-config key in a theme as well as your
+sites.
+
+This change is being made to make it easier for theme authors to produce themes
+that rely on complex behavior while enabling users who have different skill sets
+to consume and modify those themes. Theme authors will benefit from the ability
+to abstract logic, UI elements, and other Gatsby idioms into multiple packages
+which can be consumed as a single theme. Child themes (and themes as a whole)
+will improve the ability of people with different skill sets to create
+performant, beautiful sites. A few that I had in mind while building out child
```

Comment:
Would it be appropriate to add "accessible" to "performant, beautiful" since that is a goal for Gatsby themes?